lib/vanagon/common/pathname.rb

```diff
@@ -26,17 +26,18 @@ class Pathname
       # @param [String, Integer] mode the UNIX Octal permission string to use when this file is archived
       # @param [String, Integer] owner the username or UID to use when this file is archived
       # @param [String, Integer] group the groupname or GID to use when this file is archived
-      # @param [Boolean] config mark this file as a configuration file, stored as private state
+      # @param [Boolean] options => {config} mark this file as a configuration file, stored as private state
       #   and exposed through the {#configfile?} method.
       # @return [Vanagon::Common::Pathname] Returns a new Pathname instance.
-      def initialize(path, mode: nil, owner: nil, group: nil, config: false)
+      def initialize(path, mode: nil, owner: nil, group: nil, options: {})
```

Comment:
I'm weary of introducing the Options hash pattern into a series of named parameters. Was this done because `rubocop` hollered at you?